C++/C++.sublime-syntax

```diff
@@ -51,9 +51,9 @@ contexts:
     - match: (?={{identifier}}\s*<)
       branch_point: function-template
       branch: [function-template, consume-ident]
-    - match: \b{{identifier}}\b
+    - match: '{{identifier}}'
```

Comment:
Was there a problem with the word breaks?

Reply:
no, but {{identifier}} already has word breaks so i thought it was redundant